lib/src/Base/IterativeStat/openturns/IterativeThresholdExceedance.hxx

```diff
@@ -0,0 +1,95 @@
+//                                               -*- C++ -*-
+/**
+ *  @brief IterativeThresholdExceedance
+ *
+ *  Copyright 2005-2021 Airbus-EDF-IMACS-ONERA-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#ifndef OPENTURNS_ITERATIVETHRESHOLDEXCEEDANCE_HXX
+#define OPENTURNS_ITERATIVETHRESHOLDEXCEEDANCE_HXX
+
+#include "openturns/IterativeAlgorithmImplementation.hxx"
+#include "openturns/Point.hxx"
+
+BEGIN_NAMESPACE_OPENTURNS
+
+class OT_API IterativeThresholdExceedance
+  : public IterativeAlgorithmImplementation
+{
+  CLASSNAME
+
+public:
+
+  /** Constructor */
+  explicit IterativeThresholdExceedance(const UnsignedInteger dimension = 1, const Scalar thresholdValue = 0.0);
+
+  /** Virtual constructor */
+  IterativeThresholdExceedance * clone() const override;
+
+  /** Increment method from a Point */
+  virtual void increment(const Point & newData) override;
```

Comment:
no need to specify both virtual+override is useless, just override is ok

Reply:
OK